components/toast/toast.html

```diff
@@ -0,0 +1,126 @@
+            </li>
+            <li class="component-links">
+                <a href="../avatar/avatar.html">Avatar</a>
+            </li>
+            <li class="component-links">
+                <a href="../badge/badge.html">Badge</a>
+            </li>
+            <li class="component-links">
+                <a href="../button/button.html">Button</a>
+            </li>
+            <li class="component-links">
+                <a href="../card/card.html">Card</a>
+            </li>
+            <li class="component-links">
+                <a href="../chips/chips.html">Chips</a>
+            </li>
+            <li class="component-links">
+                <a href="../dialog/dialog.html">Dialog</a>
+            </li>
+            <li class="component-links">
+                <a href="../grid/grid.html">Grid</a>
+            </li>
+            <li class="component-links">
+                <a href="../image/image.html">Image</a>
+            </li>
+            <li class="component-links">
+                <a href="../input/input.html">Input</a>
+            </li>
+            <li class="component-links">
+                <a href="../list/list.html">List</a>
+            </li>
+            <li class="component-links">
+                <a href="../modal/modal.html">Modal</a>
+            </li>
+            <li class="component-links">
+                <a href="../navigation/navigation.html">Navigation</a>
+            </li>
+            <li class="component-links">
+                <a href="../rating/rating.html">Rating</a>
+            </li>
+            <li class="component-links">
+                <a href="../text-utilities/text-utilites.html">Text Utilites</a>
+            </li>
+            <li class="component-links">
+                <a href="../toast/toast.html">Toast</a>
+            </li>
+        </ul>
+    </nav>
+
+    <main class="comp-display">
+        <h1 class="comp-header">Toast</h1>
+        <p class="comp-description">
+            Toasts are lightweight notifications designed to mimic the push 
+            notifications that have been popularized by mobile and desktop 
+            operating systems. They’re built with flexbox, so they’re easy 
+            to align and position.
+        </p>
+        <h2 class="sub-comp-header">Simple Toast</h2>
+
+        <div class="comp-disp-container">
+            <div class="toast">
+                <div class="toast-heading">
+                    <h2 class="sub-comp-header">Simple Toast</h2>
+                    <h2 class="sub-comp-header">&times</h2>    
+                </div>
+                <div class="toast-text">
+                    <p>    
+                    The toast component is like an alert box that is only shown 
+                    for a couple of seconds when something happens (i.e. when the 
+                    user clicks on a button, submits a form, etc.).
+                    </p>
+                </div>
```

Comment:
this div is not necessary i feel, as only p tag is inside it. Directly give class to p itself.

Reply:
Done